extraTs/utils/validation.ts

```diff
@@ -0,0 +1,30 @@
+import{validateEmail} from './helpers'
+
+function validateUser(users){
```

Comment:
users type and function return type is missing.

Reply:
done